test/validators/wizards-v1/universe/universe-validator-v1.ts

```diff
@@ -59,4 +59,54 @@ export class UniverseValidatorV1 {
       .to.emit(universeRegistry, IUNIVERSE_REGISTRY_EVENTS.UNIVERSE_CREATED)
       .withArgs(universeId, expectedUniverseParams.name, expectedUniverseParams.paymentTokens);
   }
+
+  public static async validateUniverseNameChangedProperly(
```

Comment:
Those new functions could be inside universe-registry-validator, since they are not attached to particular Wizard version!